index.html

```diff
@@ -234,6 +226,35 @@ <h3>BrowserCaptureMediaStreamTrack extension</h3>
         </dl>
       </section>
     </section>
+    <section id="applying-the-restriction-transformation">
+      <h2><dfn>Applying the restriction transformation</dfn></h2>
+      <p>
+        When it's time to produce <var>frame</var> for a video track <var>videoTrack</var> that is
+        [=restricted=] to a given target <var>cropTarget</var>, execute the following algorithm:
+      </p>
+      <ol>
+        <li>
+          <!-- TODO(eladalon1983): Fix linkability of slots in the mediacapture-region spec. -->
+          If <var>cropTarget</var>.[[\Element]] does not represent an element that's in the DOM of
```

Comment:
> I can't find a definition for it in any W3C spec. This may be challenging because specs try to steer clear of defining the side effects of garbage collection (or memory management in general).

We had some lively debate over that back in https://github.com/w3c/mediacapture-region/pull/24, including discussion of garbage collection. I also [asked for precedents](https://github.com/w3c/mediacapture-region/pull/24#issuecomment-1081962074) and worried about garbage collection. But Youenn and Jan-Ivar did not share my concerns. (Not all of these discussions are captured on GitHub.)

> Handling of pseudo-elements should be discussed

I've not previously known about those. Is [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) what you mean?

> The element should be connected to the DOM

Giving it a second look, I think the PR captures the possibility of the element (1) being connected to the DOM, (2) being removed either before capture starts or during capture, and (3) being reconnected to the DOM.

Reply:
> But Youenn and Jan-Ivar did not share my concerns.

I'm glad that there's consensus around this.  The spec would be better if there was a definition of _weakRef_ somewhere.  I will file a followup issue.

> I've not previously known about those. Is this what you mean?

Sorry, I meant elements in the [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot).  I will file an issue to follow up.